src/components/ebay-button/index.js

```diff
@@ -47,6 +47,7 @@ function getTemplateData(state, input) {
     }
 
     model.tag = tag;
+    model.type = input.type;
```

Comment:
We keep most of the logic in the renderer, so let's do 
`model.type = input.type || 'button'`

Reply:
https://github.com/cestdiego/ebayui-core/commit/3aa5f90